.github/workflows/vkd3d.yml

```
@@ -167,370 +167,64 @@ jobs:
-                  print(f"  - {e}")
-              sys.exit(1)
-          PYSCRIPT
-          python3 patch.py
-
-      - name: Apply State Blend Fix
+          echo "$PWD/glslang/bin" >> $GITHUB_PATH
+          echo "GLSLang version:"
+          $PWD/glslang/bin/glslangValidator --version 2>/dev/null || $PWD/glslang/bin/glslang --version 2>/dev/null || true
+
+      - name: Clone Source
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:4:48: Double quote to prevent globbing and word splitting

(shellcheck)